ParisBNF/abb/BNFabb149.xml

```diff
@@ -0,0 +1,482 @@
+
+                                   <item xml:id="e2">
+                                     <locus target="#99r"/>
+                                       <desc>Omitted passage added in the upper margin,
+                                         its place in the text indicated with
+                                         <foreign xml:lang="am">tamallas-sign</foreign> (cross).</desc>
+                                       <q xml:lang="gez">ብርዕ፡ ዘተፈትነ፡</q>
+                                       </item>
+                         </list>
+                       </additions>
+                     </physDesc>
+                     </msPart>
+                   </msPart>
+
+                   <msPart xml:id="p3">
+                      <msIdentifier/>
+                      <msContents>
+                         <summary/>
+                       <msItem xml:id="p3_i1.1">
+                       <locus target="#105"/>
+                       <locus from="108r" to="109v"/>
+                       <locus target="#110"/>
+                       <title type="incomplete" ref="LIT2509Weddas"/>
+                       <incipit xml:lang="gez">ር፡ ሶቤሃ፡ ርሰነ፡ መለኮት፡ ውሉድ፡ ወሉድ፡ እማእላፍ፡ ርእሱ፡ ወርቅ፡ ቂፋዝ፡ </incipit>
+                       <explicit xml:lang="gez"></explicit>
+                       <note>Musical notation above the written lines of the main text.</note>
+                         </msItem>
+                   </msContents>
+                   <physDesc>
+                     <objectDesc form="Codex">
+                        <supportDesc>
+                           <support>
+                              <material key="parchment"/>
+                           </support>
+                           <extent>
+                              <measure unit="leaf">4</measure>
+                              <locus target="#105 #107 #108 #110"/>
+                              <dimensions type="outer" unit="mm">
+                                 <height cert="low">130</height>
+                                 <width cert="low">70</width>
+                                 </dimensions>
+                              </extent>
+                        </supportDesc>
+
+                        <layoutDesc>
+                           <layout columns="1" writtenLines="30">
+                              <note>Number of characters per line: 22, <locus target="#105v"/>.</note>
+                              <note corresp="#textarea #margin">Data on text area and margin
+                                dimensions taken from <locus target="#105r"/>.</note>
+                                <note></note>
+                              <dimensions unit="mm" xml:id="textarea3">
+                                 <height cert="low">120</height>
+                                 <width cert="low">70</width>
+                                     </dimensions>
+                              <dimensions type="margin" unit="mm" xml:id="margin3">
```

Comment:
if no measures are provided, the empty elements should be deleted

Reply:
I will try to fill later